cg/meta/workflow/raredisease.py

```diff
@@ -29,6 +29,16 @@ def __init__(
     def root(self) -> str:
         return self.config.raredisease.root
 
+    def write_managed_variants(self, case_id: str, content: list[str]) -> None:
+        """Write the managed variants to case dir."""
+        out_dir = Path(self.root, case_id)
+        out_dir.mkdir(parents=True, exist_ok=True)
+        WriteFile.write_file_from_content(
+            content="\n".join(content),
+            file_format=FileFormat.TXT,
+            file_path=Path(out_dir, f"managed_variants{FileExtensions.VCF}"),
+        )
```

Comment:
This function is exact same as in mip and does not really require a self, the root dir can be passed on as parameter instead. Can this function be extracted to a "utils" module instead, can be done by a caller inside the api I think?

Reply:
This could also work

Reply:
Done